Kitodo/src/main/java/de/sub/goobi/forms/LoginForm.java

```diff
@@ -296,8 +300,24 @@ public void setPassword(String password) {
         this.password = password;
     }
 
+    /**
+     * Gets current authenticated User.
+     *
+     * @return
+     *      The User.
+     */
     public User getMyBenutzer() {
-        return this.myBenutzer;
+        if (myBenutzer != null) {
+            return this.myBenutzer;
+        } else {
+            try {
+                myBenutzer = serviceManager.getUserService().getAuthenticatedUser();
+                return this.myBenutzer;
+            } catch (DAOException e) {
+                Helper.setFehlerMeldung(e);
+            }
+        }
+        return null;
```

Comment:
Is returning `null` a good solution? Should not a exception be raised if this user could not be found / retrieved?

Reply:
A null check is performed on frontend. In case we add a exception here, this would be fired every time a user loads the main page first time.
I just moved the return to catch block to make it more readable.